eu.usdx.UltraStarDeluxe.yaml

```
@@ -0,0 +1,180 @@
+id: eu.usdx.UltraStarDeluxe
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '22.08'
+separate-locales: false
+rename-desktop-file: ultrastardx.desktop
+rename-icon: ultrastardx
+command: ultrastardx
+
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --share=ipc
+- --socket=pulseaudio
+- --device=all
+- --filesystem=~/.java/.userPrefs/PortMidi:ro
+- --persist=.ultrastardx
+
+cleanup:
+- /include
+- /lib/cmake
+- /lib/pkgconfig
+- "/lib/*.so"
+- "/lib/*.la"
+- setup_vars_opencv4.sh
+
+add-extensions:
+  eu.usdx.UltraStarDeluxe.Songs:
+    directory: share/ultrastardx/songs
+    subdirectories: true
+    no-autodownload: true
+
+  eu.usdx.UltraStarDeluxe.Themes:
+    directory: theme-extensions
+    merge-dirs: share/ultrastardx/themes
+    subdirectories: true
+    no-autodownload: true
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.freepascal
+
+build-options:
+  append-path: /usr/lib/sdk/freepascal/bin
+  env:
+   - FPCDIR=/usr/lib/sdk/freepascal/fpcsrc
+
+modules:
+- name: opencv
+  buildsystem: cmake-ninja
+  builddir: true
+  build-options:
+    config-opts:
```

```suggestion
  config-opts:
```

I should be at the module level

---

fixed